src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -250,34 +248,6 @@ export const updateEnvironmentsInLocalStorage = (
     );
 };
 
-export const getSelectedEnvironmentUrlFromLocalStorage = (
-    localStorageKey: string = SelectedEnvironmentLocalStorageKey
-) => {
-    try {
-        return (JSON.parse(
-            localStorage.getItem(localStorageKey)
-        ) as ADTSelectedEnvironmentInLocalStorage)?.appAdtUrl;
-    } catch (error) {
-        console.error(error.message);
-        return null;
-    }
-};
-
-export const updateSelectedEnvironmentInLocalStorage = (
-    selectedEnvironment: string | IAzureResource,
-    localStorageKey: string = SelectedEnvironmentLocalStorageKey
-) => {
-    localStorage.setItem(
-        localStorageKey,
-        JSON.stringify({
-            appAdtUrl: getResourceUrl(
-                selectedEnvironment,
-                AzureResourceTypes.DigitalTwinInstance
-            )
-        })
-    );
-};
-
 export const getStorageAccountOptionsFromLocalStorage = () => {
```

Comment:
please define types for anything being returned from local storage. Then use that same type when storing the value to give us some safety.

Reply:
agreed, the local storage state should be typed and validated when read and written

Reply:
fixed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/d977cb41f7ee0b9a7ee66c8307366112358fb27c#diff-9a856bccecc07dd21c6fb673efb9e1b33bce3bcbd41c9099d49cad741f8e958eR257